[blog] Discord announcement blog (#38258)
mui/material-ui: docs/pages/blog/2023-discord-announcement.md

---
title: 'MUI is now on Discord!'
description: Come join our community to engage in lively discussions, share your projects, and interact with the MUI team
date: 2023-08-02T00:00:00.000Z
authors: ['richbustos']
tags: ['News']
card: true
---

> <a href="https://mui.com/r/discord "><img src="/static/blog/2023-discord-announcement/discord.png" width="1200" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>

Reviewer: ```suggestion
<a href="https://mui.com/r/discord"><img src="/static/blog/2023-discord-announcement/discord.png" width="1200" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>
```

---
title: 'MUI is now on Discord!'
description: Come join our community to engage in lively discussions, share your projects, and interact with the MUI team
date: 2023-08-02T00:00:00.000Z
authors: ['richbustos']
tags: ['News']
card: true
---

> <a href="https://mui.com/r/discord"><img src="/static/blog/2023-discord-announcement/discord.png" width="1200" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>

Reviewer 2: The width and height are wrong, there is a layout shift when opening the page. The best practice in our case is to use the exact image dimension `width="2560" height="1280"` at least to have the same ratio.

Author: The exact image is`width="1280" height="640"` now.

> Creating a place where we can interact with our community has been our priority for this year.

Reviewer 3: Rearranging for tidier grammar.

```suggestion
This year, one of our highest priorities has been to create a central hub for the MUI community.
```

Author: i like it!

> Majority of our discussions and interactions with you have been through our [Twitter](https://twitter.com/mui_hq), [GitHub](https://github.com/mui), and [marketing page](https://mui.com/).

Reviewer 2: ```suggestion
Majority of our discussions and interactions with you have been through our [Twitter](https://twitter.com/MUI_hq), [GitHub](https://github.com/mui), and [marketing page](https://mui.com/).
```

> Majority of our discussions and interactions with you have been through our [Twitter](https://twitter.com/MUI_hq), [GitHub](https://github.com/mui), and [marketing page](https://mui.com/).

Reviewer 3: Let's underscore the "scattered" nature of our community pre-Discord; drop "marketing page" (that's an internal term that won't resonate with readers) and add LinkedIn and SO.

```suggestion
Up until now, our community has been scattered across discussions and interactions on [Twitter](https://twitter.com/MUI_hq), [LinkedIn](https://linkedin.com/company/mui), [GitHub](https://github.com/mui), and [Stack Overflow](https://stackoverflow.com/questions/tagged/material-ui).
```

> That's why we have officially launched our [Discord Server](https://mui.com/r/discord/) and would love for you to be part of our community!

Reviewer 3: ```suggestion
That's why we've officially launched our [Discord Server](https://mui.com/r/discord/)—and we'd love for you to be a part of our community!
```

> Come join to engage in lively discussions, share your projects, and interact with the MUI team & others!

Reviewer 3: Avoid ampersands unless there's a character limit (Twitter); let's describe these "others" in more detail.

```suggestion
Come join to engage in lively discussions, share your projects, and interact with the MUI team as well as developers just like you from all around the world.
```

## Why you should join our Discord Server?

> There are a plethora of reasons why you should join, but ultimately, it came down to 3 things:

Reviewer 3: ```suggestion
There are a plethora of reasons why you should join, but here are the three biggest benefits:
```

1. **Real-time communication with our community**:

>    As mentioned above, our communication with the community was quite slow.

Reviewer 3: ```suggestion
   As mentioned above, our communication with the community has often been slow and scattered across too many different platforms.
```

>    Members would either file an issue on Github, tweet us, or send us a message directly on our website.

Reviewer 3: "Members" feels like an odd way to describe developers who use MUI products. 

```suggestion
   Developers are frequently unsure whether to open an issue on Github, tweet at us, or send us an email when they have a question or want to share feedback.
```

Author: ahh yeah, I get your point. Was hesitant to specify "developers" because there could also be designers or product managers who use our products.

I guess in a sense, those roles are also developers. 

Will change all "members" to "developers"

   _Discord provides real-time chat capabilities_, allowing you to communicate and collaborate with us and other members, instantly.

>    What's a better way to get a real-time response from the MUI team and MUI users?!

Reviewer 3: ```suggestion
   What's a better way to get a real-time response from the MUI team and other users?!
```

2. **Community building**:

>    Want to hear what MUI team is up to?

Reviewer 3: ```suggestion
   Want to hear what the MUI team is up to?
```

   Get the latest update with the products you're using?

>    Want to give us real-time feedback?

Reviewer 3: I would cut this line or replace it with a different point since it's just reiterating point 1 on the list.

Author: got it!

>    Heck, do you want to see if other's are facing the same issue you're having?

Reviewer 3: ```suggestion
   Heck, do you want to see if others are facing the same issue you're having?
```

>    Discord is a great place for us to get together and chat(Keep on the lookout for #voice-meetups)!

Reviewer 3: "Keep a lookout" or "Be on the lookout" - goofy idiom 🙃

```suggestion
   Discord is a great place for us to get together and chat! (Keep a lookout for #voice-meetups!)
```

3. **Networking and collaboration opportunities**:
   We want this to be a safe place where you can join, have fun, and meet new people.

>    Someone is out there using this same tool as you(_ahem_, MUI).

Reviewer 3: ```suggestion
   Someone is out there using this same tool as you (_ahem_, MUI).
```

Reviewer 3: I don't think I understand what the parenthetical means 😅

Author: well if you don't understand it then im removing it lol. 

I was trying to reference the "tool" as MUI lol. All good. removing it haha

>    We created two awesome channels to help you find work opportunities(We can create more, just tell us!)

Reviewer 3: ```suggestion
   We created two awesome channels to help you find work opportunities (and we can always create more—just let us know what you need)!
```

   1. 🎨 │ Show-Off
      - Show off your MUI projects! Get wild. Ask questions.

>         If you have MUI related questions that is blocking you from your project, ask here.

Reviewer 3: ```suggestion
        If you have MUI-related questions that are blocking you in your project, ask here.
```

Reviewer 3: Is this the best place to ask questions? Or would a user be better off asking in the product-specific channel? If I were browsing this server, I would not think to look here to answer other people's questions. I would assume people are just sharing stuff they've built.

Author: I was looking at it through the lens of "I'm working on a project that is using MUI and Im blocked. I need help and want to pay someone to do it. Ill go to freelance". 

But after reading that out loud, youre right, doesnt make sense lol.

Removing

        Someone may be facing it as well.

>         Or use this channel to simply boast 😉.

Reviewer 3: ```suggestion
        Or use this channel simply to boast. 😉
```

   2. 📝 │ Freelance
      - Looking for some help on completing your vision?
        Use this channel to find work or market yourself for work.

>         We have over 1000s of developers and designers using MUI.

Reviewer 3: ```suggestion
        There are thousands of developers, designers, and teams using MUI.
```

        You're bound to find someone to help you complete your project.

> Overall, Discord's combination of real-time communication, community building features, organizational capabilities, and developer-friendly features make it a valuable tool for our community to connect, collaborate, and grow together!

Reviewer 3: Rather than describing why Discord in general is neat, can we replace this with a call to action to join the server and start posting? Maybe we can offer a suggestion like "drop an introduction in the #lounge channel" or "show us something you've built" to give folks an idea of where to start?
